src/smart-components/request/request-detail/request-stage-transcript.js

```diff
@@ -0,0 +1,20 @@
+import React, { Fragment }  from 'react';
+import PropTypes from 'prop-types';
+import { Title } from '@patternfly/react-core';
+import StageList from './stage-list';
+
+const RequestStageTranscript = ({ request }) => {
+  console.log('Stages:', request ? request.stages : []);
+  return (<Fragment>
+    <Title size="sm" style={ { paddingLeft: '5%' } }> Stage(s) transcript </Title>
```

Comment:
I might be wrong on this but i think PF is using static spacing values (8, 16, 32 and 64)